application/templates/backup.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if (.Values.backup).enabled }}
+apiVersion: velero.io/v1
+kind: Backup
+metadata:
+  name: {{ printf "%s-backup" .Values.applicationName | trunc 63 | quote }}
+  namespace: {{ .Values.backup.namespace | quote }}
+spec:
+  includedNamespaces:
+    - {{ printf "%s" .Release.Namespace | quote }}
+  snapshotVolumes: {{ .Values.backup.snapshotVolumes }}
+  storageLocation: {{ .Values.backup.storageLocation | quote }}
+  ttl: {{ .Values.backup.ttl }}
```

Comment:
@ayeshasiddiqa161 we should always set default value for this so, user doesn't have to set always